public/dev/index.html

```diff
@@ -7,81 +6,18 @@
-        <link rel="stylesheet" href="%PUBLIC_URL%/index.css" />
-
-        <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
-
-      Unlike "/favicon.ico" or "favicon.ico", "%PUBLIC_URL%/favicon.ico" will
-      work correctly both with client-side routing and a non-root public URL.
-      Learn how to configure a non-root public URL by running `npm run build`.
-    -->
-
-        <!-- Hotjar Tracking Code for https://www.nav.no/ -->
-        <script>
-            (function(h, o, t, j, a, r) {
-                h.hj =
-                    h.hj ||
-                    function() {
-                        (h.hj.q = h.hj.q || []).push(arguments);
-                    };
-                h._hjSettings = { hjid: 118350, hjsv: 6 };
-                a = o.getElementsByTagName('head')[0];
-                r = o.createElement('script');
-                r.async = 1;
-                r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
-                a.appendChild(r);
-            })(
-                window,
-                document,
-                'https://static.hotjar.com/c/hotjar-',
-                '.js?sv='
-            );
-        </script>
-
-        <!-- Google Tag Manager -->
-        <script>
-            (function(w, d, s, l, i) {
-                w[l] = w[l] || [];
-                w[l].push({
-                    'gtm.start': new Date().getTime(),
-                    event: 'gtm.js',
-                });
-                var f = d.getElementsByTagName(s)[0],
-                    j = d.createElement(s),
-                    dl = l != 'dataLayer' ? '&l=' + l : '';
-                j.async = true;
-                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
-                f.parentNode.insertBefore(j, f);
-            })(window, document, 'script', 'dataLayer', 'GTM-PM9RP3');
-        </script>
-        <!-- End Google Tag Manager -->
-
         <title>Avtale om arbeidstrening</title>
+        {{{NAV_SCRIPTS}}} {{{NAV_MENU_RESOURCES}}} {{{NAV_STYLES}}}
     </head>
-
     <body>
         <noscript> You need to enable JavaScript to run this app. </noscript>
-        <div id="root" class="tiltaksgjennomforing"></div>
-        <!--
-  This HTML file is a template.
-  If you open it directly in the browser, you will see an empty page.
-
-  You can add webfonts, meta tags, or analytics to this file.
-  The build step will place the bundled scripts into the <body> tag.
-
-  To begin the development, run `npm start` or `yarn start`.
-  To create a production bundle, use `npm run build` or `yarn build`.
--->
+        {{{NAV_HEADING}}}
+        <div id="Tiltak"></div>
+        <div id="root" className="tiltaksgjennomforing"></div>
+        {{{NAV_FOOTER}}}
         <script type="text/javascript" src="/bundle.js"></script
```

Comment:
Det ser ut til at eneste forskjellen mellom dev/index.html og index.html er disse to linjene:
```
<script type="text/javascript" src="/bundle.js"></script
><script type="text/javascript" src="/static/js/main.chunk.js"></script>
```
Er det nødvendig å ha to forskjellige filer bare på grunn av det? Kan det ikke løses med mustache?